src/managers/run-strategies/multiplayer-strategy.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+var classFrom = require('../../util/inherit');
+
+var IdentityStrategy = require('./identity-strategy');
+var StorageFactory = require('../../store/store-factory');
+var WorldApiAdapter = require('../../service/world-api-adapter');
+var keyNames = require('../key-names');
+var _pick = require('../../util/object-util')._pick;
+
+var defaults = {
+    store: {
+        synchronous: true
+    }
+};
+
+var Strategy = classFrom(IdentityStrategy, {
+
+    constructor: function (runService, options) {
+        this.runService = runService;
+        this.options = $.extend(true, {}, defaults, options);
+        this._store = new StorageFactory(this.options.store);
+    },
+
+    reset: function () {
+        throw new Error('Not supported. Reset should not be called from the end-user side');
+    },
+
+    getRun: function () {
+        var session = JSON.parse(this._store.get(keyNames.EPI_SESSION_KEY) || '{}');
+        var curUserId = session.userId;
+        var opt = $.extend({
+            account: session.account,
+            project: session.project,
+            group: session.groupName
+        }, _pick(this.options, ['account', 'project', 'group']));
+
+        var dtd = $.Deferred();
+
+        if (!curUserId) {
+            return dtd.reject({ statusCode: 400, error: 'We need an authenticated user to join a multiplayer world. (ERR: no userId in session)' }, session).promise();
+        }
+
+        var worldApi = new WorldApiAdapter(opt);
+
+        var restoreInitRun = function (worlds, msg, xhr) {
+            if (!worlds || !worlds.length) {
+                return dtd.reject({ statusCode: 404, error: 'The user is not in any world.' }, { options: opt, session: session, xhr: xhr });
```

Comment:
The userId is actually part of the session object, that also includes the account/project/group